utils/selenium_utils.py

```diff
@@ -40,6 +40,16 @@ def yes_amazon_image():
     options.add_experimental_option("prefs", prefs)
 
 
+def no_walmart_image():
```

Comment:
I was using 2captcha.com's integration. For $3 you get 1000 solves and it seemed like a small price to pay. I couldn't get the site to accept the captcha response though. All of the work is being done under this in walmart.py:

`def defeat_captcha(self, url):`

I started with amazon.py as a template and started hacking in what I needed for Walmart.  There may be bits and pieces that can be removed to clean it up. As for the captcha, I have an open case with 2captcha. If I hear back with a fix, I will update the code.

Reply:
I meant it more in the sense of, what is the method of captcha solving? Does it need images to load? I'm asking because I don't know how 2captcha.com does it - if they don't need images to load, then putting in the --no-images flag is fine. If that won't work, no sense in putting in the option, since walmart seems to love putting in captchas

Reply:
Hey agmeadows regarding the ReCaptcha issue, it could be that Google has flagged your IP and it's not accepting any solved captchas from that IP, at least what 2captcha are saying here https://2captcha.com/blog/google-doesnt-accept-recaptcha-answers